include/nuttx/sched.h

```diff
@@ -861,6 +863,12 @@ EXTERN clock_t g_premp_max[CONFIG_SMP_NCPUS];
 EXTERN clock_t g_crit_max[CONFIG_SMP_NCPUS];
 #endif /* CONFIG_SCHED_CRITMONITOR_MAXTIME_CSECTION >= 0 */
 
+/* g_running_tasks[] holds a references to the running task for each cpu.
```

Comment:
```suggestion
/* g_running_tasks[] holds a reference to the running task for each cpu.
```

Also, maybe make CPU uppercase?